benchmarks/python/comparative/bench_mlx.py

```diff
@@ -99,6 +99,48 @@ def relu(x):
     mx.eval(y)
 
 
+def leaky_relu(x):
+    y = x
+    for i in range(100):
+        y = mx.maximum(0.1 * y, y)
```

Comment:
I see you are using the implementation to benchmark. I would use `nn.leaky_relu`, `nn.elu` etc. I realize it may have been confusing that we use `mx.maximum(x, 0)` for `relu` above, feel free to update that too, the only reason for that was that this has been there before we had `nn.relu` as a function :-) .

Reply:
Also to make it a bit easier for yourself and for the future, I would write one `activation` function that gets the activation name as a string and then the function via `getattr(nn, act_name)` so you don't have to write the same thing 5 times.

Reply:
> I see you are using the implementation to benchmark. I would use `nn.leaky_relu`, `nn.elu` etc. I realize it may have been confusing that we use `mx.maximum(x, 0)` for `relu` above, feel free to update that too, the only reason for that was that this has been there before we had `nn.relu` as a function :-) .

Ahh I see, noted, will change this in the next commit